compose/ui/ui-test/src/uikitMain/kotlin/androidx/compose/ui/test/InputDispatcher.uikitMain.kt

```diff
@@ -20,13 +20,34 @@ package androidx.compose.ui.test
 import androidx.compose.ui.input.key.Key
 import androidx.compose.ui.input.key.KeyEvent
 import androidx.compose.ui.input.key.KeyEventType
+import org.jetbrains.skiko.SkikoInputModifiers
+import org.jetbrains.skiko.SkikoKey
+import org.jetbrains.skiko.SkikoKeyboardEvent
+import org.jetbrains.skiko.SkikoKeyboardEventKind
 
 /**
  * The [KeyEvent] is usually created by the system. This function creates an instance of
  * [KeyEvent] that can be used in tests.
  */
 internal actual fun keyEvent(
     key: Key, keyEventType: KeyEventType, modifiers: Int
-): KeyEvent = TODO()
+): KeyEvent {
+    return return KeyEvent(
+        SkikoKeyboardEvent(
+            key = SkikoKey.values().firstOrNull {
```

Comment:
```suggestion
// TODO replace  `enum class SkikoKey` by `class SkikoKey` in skiko, change code here to `SkikoKey(it.platformKeyCode)`
            key = SkikoKey.values().firstOrNull {
```


I think, we need to change `enum class SkikoKey` to just `class SkikoKey` (in the future), because we can encounter key codes, of which we don't know about, and which we still need to pass to the user.